pkgs/servers/x11/xorg/default.nix

```diff
@@ -1717,11 +1717,11 @@ lib.makeScope newScope (self: with self; {
   }) {};
 
   xf86inputlibinput = callPackage ({ stdenv, pkg-config, fetchurl, xorgproto, libinput, xorgserver }: stdenv.mkDerivation {
-    name = "xf86-input-libinput-1.0.0";
+    name = "xf86-input-libinput-1.0.1";
```

Comment:
Any particular reason we don't move to pname/version here?

Reply:
Someone would need to update https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/x11/xorg/generate-expr-from-tarballs.pl.